examples/abitest/client/grpc_test_server.cc

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "examples/abitest/client/grpc_test_server.h"
+
+#include "asylo/util/logging.h"
+
+using ::oak::examples::abitest::GrpcTestRequest;
```

Comment:
(nit) It seems we tend not to use the leading `::` in other places, anyways I don't particularly care either way.

Reply:
At the moment all the `using` statements in `examples/*/client/*.cc` have the leading `::`, so there's at least partial consistency&hellip;